### DISCUSSION TITLE
next-pwa: API data not updating every-time. API not called.

### DISCUSSION THREAD

**Discussion by 2manoj1** [vercel/next.js#11292] [Help]
- cache issue. No Update after next-pwa implementation. 
- Need every time api should hit get latest data and update.
- After clear cache its called api.

Can anyone help how I fix this issue with next-pwa?

**Comment by woro83c** [comment]
I have a similar issue, but unrelated to next-pwa. We have the following headers set:

```json
{
  "source": "/(.*)",
  "headers": [
    {
      "key": "Cache-Control",
      "value": "s-maxage=1, stale-while-revalidate"
    }
  ]
}
```

But then we're getting a stale response every time we hit the API (at least for quite a while) - shouldn't we be getting a fresh response?

**Reply by 2manoj1** [reply]
`source` is not allowed in now.json when using Now 2.0 – please remove it.

**Reply by woro83c** [reply]
pretty sure it is -- see here:
https://zeit.co/docs/configuration#project/headers

**Comment by 2manoj1** [comment]
not able to clear cache after so many redeploy. everyone giving feedback data not updating.
